mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundUTXO.tsx

```diff
@@ -61,7 +59,7 @@ export function PlaygroundUTXO (): JSX.Element {
             <PlaygroundAction
               onPress={async () => {
                 const address = await getActiveAddress()
-                fetchTokens(whaleApiClient, address, dispatch, logger)
+                dispatch(fetchTokens({ client: whaleApiClient, address }))
```

Comment:
maybe rename this to client? so it's consistent

Reply:
Will make changes